### PR TITLE
index file support

### PR DESCRIPTION
**Pull request by lukasheinrich**
this PR adds support for a new type of configuration file, an "index" file that allows to add multiple analyses
in one go to the catalogue.

This is useful for the organization of meta-analyses, like the pMSSM scan.

cc @matthewfeickert @danikam

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
   include_package_data = True,
   install_requires = [
     'click',
+    'jsonschema',
```
Comment (matthewfeickert) [inline setup.py:23]: @lukasheinrich if possible can you add a known lower bound for `jsonschema` here?

**Review by matthewfeickert — APPROVED** [review]
Minor comment, but LGTM

**Comment by danikam** [comment]
Thanks @lukasheinrich! No technical concerns from my end, just a question - I'm still not quite following how the 'tags' fields would be used in practice. Would you happen to have an example config file which illustrates the indexing and tags?

File: setup.py (hunk 1 above)
Comment (lukasheinrich) [reply setup.py:23]: `pip freeze` gives me `jsonschema==3.2.0` but idk whether lower bounds work too

**Comment by lukasheinrich** [comment]
hi @danikam  - this was just to be able to collect grrouops of analysis (e.g. all analyses related to pmssm)

File: setup.py (hunk 1 above)
Comment (lukasheinrich) [reply setup.py:23]: i'll mark this as a new issue to figure out a sane setup.py

**Pull request merged**